packages/datadog-plugin-mysql/src/index.js

```diff
@@ -55,4 +56,14 @@ class MySQLPlugin extends Plugin {
   }
 }
 
+function getServiceName (tracer, config, dbConfig) {
+  if (typeof config.service === 'function') {
+    return config.service(dbConfig)
```

Comment:
Is the [documentation](https://datadoghq.dev/dd-trace-js/interfaces/plugins.mysql.html) generated from this repo? If so, would be good to update it to reflect that `service` can be a function.

Reply:
Yes, as long as the API docs have been updated (which is the case for this PR) then the next time we release the documentation will be updated.